server/routerlicious/packages/local-server/webpack.config.js

```diff
@@ -23,7 +23,12 @@ module.exports = {
         ]
     },
     resolve: {
-        extensions: [ '.tsx', '.ts', '.js' ]
+        extensions: [ '.tsx', '.ts', '.js' ],
+        fallback: {
+            // To polyfill buffer, use: require.resolve("buffer/")
```

Comment:
if you do symlink:full do the examples' tests pass? Many have jest tests that will launch chrome with local server and valid the example works

Reply:
As far as I can tell, the bundle this produces it not used (its not the entry point to the package, and its not imported by the entry point, not is it imported directly by the users of this module either). Those examples are already on webpack 5, have separate configs, and I tested their polyfill settings by running their tests and running them interactively when I updated them to webpack 5.

Anyway, with symlink full I ran `npx lerna run --scope "@fluid-example/*" test:jest --concurrency=1` which passed, which I think answers your question.